src/common_swagger_api/schema/apps.clj

```diff
@@ -137,6 +144,16 @@
           SortFieldOptionalKey
           (describe (apply enum AppSearchValidSortFields) SortFieldDocs)}))
 
+(defschema QualifiedAppId
+  {:system_id SystemId
+   :app_id    StringAppIdParam})
+
+(defschema AppDeletionRequest
+  (describe
+    {:app_ids                              (describe [QualifiedAppId] "A List of qualified app identifiers")
```

Comment:
`AppDeletionRequest` used to be merged with a separate `AppIdList` schema, but nothing else was using `AppIdList`, so I just inlined here.